index.html

```diff
@@ -7,7 +7,7 @@
     <title>Hello Microverse</title>
```

Comment:
- Let's link the css file to the html file to make our styles work, you can proceed like this: `<link rel="stylesheet" hrf="./styles.css" >` :recycle: